### ISSUE TITLE
[Accessibility]: Global nav bar: Command palette item inaccessible via keyboard

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35118]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34411

### Problem description

1. Go to Sourcegraph navigation bar as an authenticated user
2. Navigate across the bar using the <kbd>Tab</kbd> key
3. Notice that the command palette item is impossible to select with the keyboard

### Expected behavior

We can select the command palette using the keyboard.

I think the root cause of this issue is that it is a `<span` with `role='button'`

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@gitstart-sourcegraph Please stop working on this, we have a duplicate issue that has been fixed by a different team: https://github.com/sourcegraph/sourcegraph/pull/37523/files. This was on the boundary between two teams ownership - sorry about this!

**Comment by umpox** [comment]
Duplicate of https://github.com/sourcegraph/sourcegraph/issues/36859 and fixed by that PR above. Closing!
